### PR TITLE
Updated the azure dependencies to compatible with Ansible 2.9.0

### PR DESCRIPTION
**Pull request by JunqiZhang0**
Updated the azure dependencies to compatible with Ansible 2.9.0

### CODE REVIEW

File: requirements-azure.txt [hunk 1]
```diff
@@ -1,36 +1,42 @@
-packaging
-requests[security]
+PyYAML
```
Comment (abraverm) [inline requirements-azure.txt]: PyYAML is ansible requirement

File: requirements-azure.txt [hunk 2]
```diff
@@ -1,36 +1,42 @@
...
-azure-mgmt-rdbms==1.4.1
 azure-mgmt-servicebus==0.5.3
 azure-mgmt-sql==0.10.0
 azure-mgmt-storage==3.1.0
 azure-mgmt-trafficmanager==0.50.0
 azure-mgmt-web==0.41.0
 azure-nspkg==2.0.0
 azure-storage==0.35.1
+cryptography
+jinja2
```
Comment (abraverm) [inline requirements-azure.txt]: Those are also ansible requirements:
https://github.com/ansible/ansible/blob/devel/requirements.txt

Comment (JunqiZhang0) [reply requirements-azure.txt]: Let me try get rid of these

**Comment by abraverm** [comment]
looks like `pip install ansible[azure]` works too, why not use that instead ?

**Comment by JunqiZhang0** [comment]
> looks like `pip install ansible[azure]` works too, why not use that instead ?

We tried this before, it only installs ansible for you, we need azure dependencies

**Comment by abraverm** [comment]
> > looks like `pip install ansible[azure]` works too, why not use that instead ?
> 
> We tried this before, it only installs ansible for you, we need azure dependencies

It worked for me today... maybe you need to install ansible first?

**Comment by 14rcole** [comment]
@junqizhang Will this PR resolve the issue I'm seeing in my tests?
```
msg": "Failed to import the required Python library (ansible[azure] (azure >= 2.0.0)) on lp-centos7-ced21096-ea72-4f0f-b4c8-da3cc89767d5-s7mj0-wzs57's Python /usr/bin/python2. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
```

**Comment by JunqiZhang0** [comment]
@14rcole yes

**Comment by samvarankashyap** [comment]
[merge]

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**